apis/python/src/tiledbsoma/util.py

```diff
@@ -91,3 +91,19 @@ def uri_joinpath(base: str, path: str) -> str:
                 parts[2] = parts[2] + "/" + path
 
     return urllib.parse.urlunparse(parts)
+
+
+def slice_to_list(sl: slice) -> List[int]:
+    """
+    TODO: COMMENT GOES HERE
```

Comment:
Please do the todo before landing :-)

Reply:
i'll do the todo before taking it out of draft ;)

#1 focus is fixing the CI error about int64 vs uint64; then neatening after that